ansible/roles/ocp-workload-3scale-demo/tasks/workload.yml

```diff
@@ -88,22 +88,15 @@
   shell: "oc project default"
 
 - name: Create the 3scale app
-  shell: "oc new-app -f https://raw.githubusercontent.com/3scale/3scale-amp-openshift-templates/{{threescale_version}}/amp/amp-eval-tech-preview.yml \
+  shell: "oc new-app -f https://raw.githubusercontent.com/jbossdemocentral/3scale-security-oidc-demo/master/support/templates/3scale-amp-template.yml \
```

Comment:
Can you please pin the versions ?
Here you are using the master branch. 

Either use a tag, or a commit ID

Reply:
tagged repo and use the same var as before